docs/api/trace.rst

```diff
@@ -8,6 +8,7 @@ Submodules
 
    trace.status
    trace.span
+   trace.ids_generator
```

Comment:
Adding it to the docs here for visibility. Also because both `span.py` and `status.py` are added and they are in the same directory level.